common/djangoapps/pipeline_js/templates/xmodule.js

```diff
@@ -4,21 +4,20 @@
 ## These files assume that several libraries are available and bound to
 ## variables in the global context, so we load those libraries with requirejs
 ## and attach them to the global context manually.
-define(["jquery", "underscore", "mathjax", "codemirror", "tinymce",
+define(["jquery", "underscore", "codemirror", "tinymce",
         "jquery.tinymce", "jquery.qtip", "jquery.scrollTo", "jquery.flot",
         "jquery.cookie",
         "utility"],
-       function($, _, MathJax, CodeMirror, tinymce) {
+       function($, _, CodeMirror, tinymce) {
     window.$ = $;
     window._ = _;
-    window.MathJax = MathJax;
+    require(['mathjax']);
```

Comment:
@andy-armstrong I will create a sandbox on this pr and ask the team here in Pakistan to do the manual testing as they always test the stage server before the release. I will provide them all the information they need to testing this scenario. @raeeschachar can do this for us.
@adampalay FYI